apps/system/js/visibility_manager.js

```diff
@@ -121,24 +121,22 @@
           this.publish('hidewindow', { type: evt.type });
         }
         break;
-      case 'mozChromeEvent':
-        if (evt.detail.type == 'visible-audio-channel-changed') {
-          this._resetDeviceLockedTimer();
-
-          if (this._normalAudioChannelActive &&
-              evt.detail.channel !== 'normal' && Service.query('locked')) {
-            this._deviceLockedTimer = setTimeout(function setVisibility() {
-              if (window.Service.query('locked')) {
-                this.publish('hidewindow',
-                  { screenshoting: false, type: evt.type });
-              }
-            }.bind(this), 3000);
-          }
```

Comment:
Same here, let's keep the `mozChromeEvent` case for now, to make sure it still works on the master branch with latest gecko.